ndbench-janusgraph-plugins/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+# Directives to determine what backend to use
+ndbench.config.janusgraph.storage.backend=cql
+
+# Common configs
+ndbench.config.janusgraph.storage.hostname=127.0.0.1
```

Comment:
So in NDBench there is a list of plugins that one can select. Here, you use localhost and that fine but it might confuse people. Shall we add at the plugin name that this is for localhost only?

Reply:
The localhost is meant to be a default vaule. Found the @Configuration annotation and I am switching to it